.github/workflows/ci.yml

```diff
@@ -36,6 +36,26 @@ jobs:
       - name: Lint the code
         run: task lint
 
+  lint-generated-project:
+    name: Lint generated project code
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
```

Comment:
I would prefer to use the latest version of this action, which is v4, seems stable enough for me.

Reply:
> The version v4 for actions/checkout seems incorrect. The latest stable version is v2. Consider using the correct version to avoid potential issues.

I would prefer to use the latest version of this action, which is v4, seems stable enough for me.